scripts/perftune.py

```diff
@@ -643,6 +643,77 @@ def __intel_irq_to_queue_idx(self, irq):
         else:
             return sys.maxsize
 
+    def __mlx_irq_to_queue_idx(self, irq):
+        """
+        Return the HW queue index for a given IRQ for Mellanox NICs in order to sort the IRQs' list by this index.
+
+        Mellanox NICs have the IRQ which name looks like
+        this:
+        mlx5_comp23
+             <bla-bla>mlxX_comp<index>
+        or this:
+        mlx4-6
+             <bla-bla>mlxX-<index>
+
+        :param irq: IRQ number
+        :return: HW queue index for Mellanox NICs and 0 for all other NICs
+        """
+        mlx_fp_irq_re = re.compile("mlx\d_comp(\d+)|mlx\d\-(\d+)")
+
+        m = mlx_fp_irq_re.search(self.__irqs2procline[irq])
+        queue=m.group(2)
+        return int(queue)
+
+    def __get_driver_name(self, iface):
```

Comment:
What is this? How is this related to Mlx support?
Move to a separate patch, please.